Netable/Example/Helpers/CustomDecodingStrategy.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CustomDecodingStrategy.swift
+//  Example
+//
+//  Created by Amy Oulton on 2022-12-14.
+//  Copyright © 2022 Steamclock Software. All rights reserved.
+//
+
+import Foundation
+import Netable
+
+struct MyCodingKey: CodingKey {
+
+    var stringValue: String
+    var intValue: Int?
+
+    init?(stringValue: String) {
+        self.stringValue = stringValue
+        self.intValue = nil
+    }
+
+    init?(intValue: Int) {
+        self.stringValue = String(intValue)
+        self.intValue = intValue
+    }
+
+}
+
+public extension JSONDecoder.KeyDecodingStrategy {
+
+    static let convertFromKebabCase = JSONDecoder.KeyDecodingStrategy.custom({ keys in
+        var modifiedKey = ""
+        let key = keys.last!
```

Comment:
This is actually how the apple docs recommended to do it, so I figured maybe it was actually desirable and I just didn't know the case where it would be! 🙃 I totally thought the exact same thing! 

I'll add in some guard statements to make it more conformant to how we would actually want to do it.

Reply:
If it's from the apple docs it's probably fine to just leave it and link to where you got it from